### ISSUE TITLE
Fill not working in WEBGL mode?

### ISSUE THREAD

**Issue by SaintPepsi** [processing/p5.js#1559]
I was starting to follow some of @shiffman 's tutorials and i've seem to have stumbled on a potential bug?

Whenever I use fill mode in 3d I get: Uncaught TypeError: Cannot read property '_applyColorBlend' of undefined p5.js:30677

My code:  

```
function setup() {
  createCanvas(500, 500, WEBGL);
}
function draw() {
  background(51);
  fill(100,100,100);
  box(300);
}
```

Am I doing something wrong or is there a different method for filling 3d objects?

**Comment by hachpai** [comment]
I had the same issue, setting back 0.5.1 version of P5js resolved it.

**Comment by indefinit** [comment]
@SaintPepsi and @hachpai, were you running the p5 build from the master branch or release?  I can't seem to replicate the error using the master branch.

**Comment by SaintPepsi** [comment]
I just discovered that p5 said there was an update.

**Comment by lmccart** [comment]
yep I think this should be fixed as of 0.5.3, published last week.
